pyarr/radarr_api.py

```diff
@@ -0,0 +1,998 @@
+from .request_api import RequestAPI
+
+
+class RadarrAPI(RequestAPI):
+    """API wrapper for Radarr endpoints."""
+
+    def construct_movie_json(
```

Comment:
should this be private as it a user would notmally just need to use add_movie without needing to construct the json first?